src/js/jsx/sections/style/ShadowList.jsx

```diff
@@ -452,15 +456,11 @@ define(function (require, exports) {
                     );
                 }, this).toList();
             } else {
-                shadowsContent = Immutable.List.of((
+                shadowsContent = (
```

Comment:
The reason of changing from list to single react component is because React require all children to have a unique key prop, if you pass in a list/array as child component.